tests/unit/utils/test_langchain_utils.py

```
@@ -0,0 +1,59 @@
+class TestLangchainUtils(unittest.TestCase):
+    def test_module_imports(self):
+        """Simple test to verify imports work"""
+        # This is a basic test to check that our module exists and can be imported
+        from codedog.utils import langchain_utils
+        self.assertTrue(hasattr(langchain_utils, 'load_gpt_llm'))
+        self.assertTrue(hasattr(langchain_utils, 'load_gpt4_llm'))
+        
+    @patch('codedog.utils.langchain_utils.env')
+    def test_load_gpt_llm_functions(self, mock_env):
+        """Test that the load functions access environment variables"""
+        from codedog.utils.langchain_utils import load_gpt_llm
```

Ruff (0.8.2)

`codedog.utils.langchain_utils.load_gpt_llm` imported but unused

Remove unused import: `codedog.utils.langchain_utils.load_gpt_llm`

(F401)